config/environments/development.rb

```diff
@@ -42,4 +42,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # The page to create a ticket is open to the world by default?
+  config.ticket_is_open_to_the_world = true
```

Comment:
[Here it is](https://github.com/ivaldi/brimir/pull/312/commits/edf760f8887fbdd871b857b8daa020b09f7d2743). Let me know what you think